<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateModel</class>
 <widget class="QDialog" name="CreateModel">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Model</string>
  </property>
  <widget class="QPushButton" name="setmodel_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Model</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="layercount_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Layer Count: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="layercount_spinBox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>380</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="layerCustomizationArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>521</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layerCustomizationWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>519</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearrseq_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear R_Seq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restore_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>39</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
  </widget>
  <widget class="QLabel" name="target_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>188</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Variable:</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainingcols_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>12</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Training Columns:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="stepfuture_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Step Future:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="stepfuture_spinbox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="steppast_label">
      <property name="text">
       <string>Step Past:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="steppast_spinbox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="forecastperiod_label">
      <property name="text">
       <string>Forecast Period:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="forecastperiod_spinbox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>135</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="optimizer_label">
      <property name="text">
       <string>Optimizer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="optimizer_combobox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>184</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="loss_label">
      <property name="text">
       <string>Loss:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="loss_combobox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="epochs_label">
      <property name="text">
       <string>Epochs:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="epochs_spinbox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="trainingcols_listwidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>35</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectalltcols_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>135</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QComboBox" name="target_combobox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deselectalltcols_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>135</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
